modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMFormInstanceRecordVersionTableReferenceDefinitionTest.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.change.tracking.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.change.tracking.test.util.BaseTableReferenceDefinitionTestCase;
+import com.liferay.dynamic.data.mapping.model.DDMFormInstance;
+import com.liferay.dynamic.data.mapping.model.DDMFormInstanceRecord;
+import com.liferay.dynamic.data.mapping.service.DDMFormInstanceRecordLocalService;
+import com.liferay.dynamic.data.mapping.test.util.DDMFormInstanceTestUtil;
+import com.liferay.portal.kernel.model.change.tracking.CTModel;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Gislayne Vitorino
+ */
+@RunWith(Arquillian.class)
+public class DDMFormInstanceRecordVersionTableReferenceDefinitionTest
+	extends BaseTableReferenceDefinitionTestCase {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		DDMFormInstance ddmFormInstance =
+			DDMFormInstanceTestUtil.addDDMFormInstance(
+				group, TestPropsValues.getUserId());
+
+		_ddmFormInstanceRecord =
+			_ddmFormInstanceRecordLocalService.addFormInstanceRecord(
+				TestPropsValues.getUserId(), group.getGroupId(),
+				ddmFormInstance.getFormInstanceId(),
+				DDMFormInstanceTestUtil.createSettingsDDMFormValues(),
+				ServiceContextTestUtil.getServiceContext());
+	}
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return _ddmFormInstanceRecordLocalService.updateFormInstanceRecord(
+			TestPropsValues.getUserId(),
+			_ddmFormInstanceRecord.getFormInstanceRecordId(), false,
+			_ddmFormInstanceRecord.getDDMFormValues(),
+			ServiceContextTestUtil.getServiceContext());
```

Comment:
Use _ddmFormInstanceRecordLocalService.updateStatus so we don't add other unrelated entities in the ctCollection.